### PR TITLE
Add fade animation and missing animation behavior

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `AnimationBehavior` <https://github.com/CommunityToolkit/Maui/issues/35>
- `FadeAnimation` <https://github.com/CommunityToolkit/Maui/issues/519>

### CODE REVIEW

**Comment by PRMerger8** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger13** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/animations/fade-animation.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    
+    var fadeAnimation = new FadeAnimation();
+    
+    await fadeAnimation.Animate(label);
+}
+```
+
+## Examples
+
+You can find an example of this animation in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/AnimationBehaviorBehaviorPage.xaml).
```
Comment (jfversluis) [inline docs/maui/animations/fade-animation.md:35]:
```suggestion
You can find an example of this animation in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/AnimationBehaviorPage.xaml).
```

File: docs/maui/animations/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: Aniations - .NET MAUI Community Toolkit
```
Comment (jfversluis) [inline docs/maui/animations/index.md:2]:
```suggestion
title: Animations - .NET MAUI Community Toolkit
```

File: docs/maui/animations/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+author: bijington
+description: The .NET MAUI Community Toolkit provides a collection of pre-built, reusable animations to make developers lives easier.
+ms.date: 07/27/2022
```
Comment (jfversluis) [inline docs/maui/animations/index.md:5]:
```suggestion
ms.date: 09/16/2022
```

File: docs/maui/animations/fade-animation.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+title: FadeAnimation - .NET MAUI Community Toolkit
+author: bijington
+description: The FadeAnimation provides the ability to animate the opacity of a VisualElement from it's original opacity, to a specified new opacity and then back to the original.
+ms.date: 07/27/2022
```
Comment (jfversluis) [inline docs/maui/animations/fade-animation.md:5]:
```suggestion
ms.date: 09/16/2022
```

File: docs/maui/behaviors/animation-behavior.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
+---
+title: AnimationBehavior - .NET MAUI Community Toolkit
+author: bijington
+description: The AnimationBehavior is a Behavior that provides the ability to animation any VisualElement it is attached to.
+ms.date: 07/27/2022
```
Comment (jfversluis) [inline docs/maui/behaviors/animation-behavior.md:5]:
```suggestion
ms.date: 09/16/2022
```

File: docs/maui/behaviors/animation-behavior.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+
+The following examples show how to add the `AnimationBehavior` to a `Label` and use the `FadeAnimation` to animate a change in opacity.
+
+### XAML
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Behaviors.AnimationBehaviorBehaviorPage">
```
Comment (jfversluis) [inline docs/maui/behaviors/animation-behavior.md:24]:
```suggestion
             x:Class="CommunityToolkit.Maui.Sample.Pages.Behaviors.AnimationBehaviorPage">
```

File: docs/maui/behaviors/animation-behavior.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+                    Scale="1"/>
+            </toolkit:AnimationBehavior.AnimationType>
+        </toolkit:AnimationBehavior>
+    </Entry.Behaviors>
+</Entry>
+```
+
+## Examples
+
+You can find an example of this behavior in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/AnimationBehaviorBehaviorPage.xaml).
```
Comment (jfversluis) [inline docs/maui/behaviors/animation-behavior.md:225]:
```suggestion
You can find an example of this behavior in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/AnimationBehaviorPage.xaml).
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
Some little fixes here!

**Review by jfversluis — APPROVED** [review]

**Pull request merged**